examples/logger.py

```diff
@@ -3,20 +3,60 @@
 """
 
 import time
+import logging
 from typing import Any, Optional, Tuple
 
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
 
 from qiwis import BaseApp
 
+
+class Signaller(QObject):
+    """Signal for LoggingHandler.
+
+    Signals:
+        signal(log): A formatted log message is emitted. 
+    """
+
+    signal = pyqtSignal(str)
+
+
+class LoggingHandler(logging.Handler):
+    """Handler for logger.
+
+    Sends a log message to connected function using emit.   
+    """
+
+    def __init__(self, slotfunc):
+        """Extended.
+
+        Connects the slotfunc to the signal.
+
+        Args:
+            slotfunc: A method addLog in class LoggerApp.
```

Comment:
Nice that you write it in a correct format, but the comment content is not appropriate.
The docstrings for public API (public classes, methods, functions, etc.) should be informative for users.
They should describe the general purpose and conditions for each function and its arguments.

In this case for `slotfunc`, your description is true but not informative for users.
Suppose that you are a user of this module. You want to use `LoggingHandler`. Would you think this docstring is helpful? It seems that it can only be used by `LoggerApp`, even if `LoggingHandler` is a public class.

From now on, please write the docstring as general as possible!

My suggestion: "A slot function for the log signal. It will be called with a log message when a log record is emitted."

Reply:
I agree. Thanks for detailed instruction. My thought was short. I will follow exact format and think in general perspective.

I fixed codes